server/test/article.spec.js

```diff
@@ -167,5 +167,13 @@ describe('Articles routes', () => {
       expect(res).to.have.status(422);
       expect(message).to.be.equal('title, body, description, tags and published fields are required');
     });
+    it('it should get all articles', async () => {
+      const res = await chai.request(app)
+        .get('/api/v1/article')
+        .set({ Authorization: `Bearer ${userToken}` });
+      const { message } = res.body;
+      expect(res).to.have.status(200);
+      expect(message).to.be.equal('Articles successfully retrieved');
+    });
   });
 });
```

Comment:
Hi @BukkyOmo does not seem you covered the test case for where no article exists

Reply:
I'm not covering for that because there will always be articles in the database, if I cater for when no article is in the database, our tests will start failing.

Reply:
Alright